config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-release-v1beta1.yaml

```diff
@@ -11,7 +11,7 @@ periodics:
   extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.1
+      base_ref: release-1.2
```

Comment:
I'm confused... didn't https://github.com/kubernetes/test-infra/pull/26049/files already update this?

Reply:
right, thanks for sharing your confusion

this update is in master (my local master was out of date)

why is github reporting the diff against master this way????

in any event nothing to see here!

Reply:
🐛

I had a feeling of déjà vu 😄